scripts/lib/install

```diff
@@ -99,17 +99,24 @@ export LANG="en_US.UTF-8"
 export LANGUAGE="en_US.UTF-8"
 
 # Check for a supported OS release.
-apt-get install -y lsb-release
-os_info="$(lsb_release --short --id --release --codename)"
-{ read -r os_id; read -r os_release; read -r os_codename; } <<< "$os_info"
+if [ -f /etc/debian_version ]; then
+    apt-get install -y lsb-release
+    os_info="$(lsb_release --short --id --release --codename)"
+    { read -r os_id; read -r os_release; read -r os_version_id; } <<< "$os_info"
```

Comment:
But that one really is a `CODENAME`, not a `VERSION_ID`, in `/etc/os-release` terminology.

Can we just skip straight to the step where we use `/etc/os-release` on Debian too? The extra 4 or 5 lines of delta are more than worth it for the reduction in complexity.

Reply:
@andersk just to make sure, does that mean that Xenial will be identified as '16.04', Bionic as '18.04', and so on?

Reply:
The problem with `os_version_id`: CentOS will be identified as '7', which is hardly a unique identifier.

Reply:
The solution is the same as before: an OS should be identified by its (`ID`, `VERSION_ID`) combination, not by either one individually.

Reply:
Additionally, changing to `os_version_id` will break this line: https://github.com/zulip/zulip/blob/1f8d21af746515d37ded52c8e1647f341be29cab/scripts/lib/install#L127 since it expects to grep the `os_codename`.

Reply:
So keep `VERSION_CODENAME` as `$os_version_codename`, just don’t use it to identify the OS. It won’t be there on CentOS, which is fine, because this line is Ubuntu-specific.

Reply:
Fixed at the 12th commit, `prod install: Use /etc/os-release for Ubuntu/Debian to get os_id, os_version_id.`.

Reply:
*11th commit.